examples/echo-py/README.md

```diff
@@ -1,4 +1,14 @@
+# Echo Python worker
+
 _echo-py_ is an example "echo" yggdrasil worker, written in Python, that uses
 _libygg_ through gobject-introspection to communicate with `yggd` over D-Bus.
 `main.py` is documented with comments demonstrating how to create and connect a
 worker using the API.
+
+## Setting up
+
+```shell
+$ sudo dnf install python3-gobject
```

Comment:
I think this needs to be integrated into `meson.build`, since it's possible to install examples. I don't know how well meson handles Python dependencies, but it should be considered.